splunkapp/generate_wazuh_splunk_app.sh

```diff
@@ -32,7 +32,7 @@ help() {
     echo "Usage: $0 [OPTIONS]"
     echo
     echo "    -b, --branch <branch>     [Required] Select Git branch or tag e.g. 3.8 or v3.8.1-7.2.3"
-    echo "    -s, --store <directory>   [Optional] Destination directory by default /tmp/splunk-app"
+    echo "    -s, --store <directory>   [Optional] Destination directory by default splunkapp/output/"
```

Comment:
I would change this to `${CURRENT_PATH}/output`. This variable is defined here: https://github.com/wazuh/wazuh-packages/blob/716e1d7c89152f343365eea9b3ee6ee9d925295a/splunkapp/generate_wazuh_splunk_app.sh#L16 and is used to get where is located the `generate_wazuh_splunk_app.sh`

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/305/commits/1dedde988334f5d9ba81861915474ca4e99b6c10